pull-request.sh

```diff
@@ -71,10 +73,22 @@ create_pull_request() {
     # Option 2: Open a new pull request
     else
         # Post the pull request
-        DATA="{\"title\":${TITLE}, \"body\":${BODY}, \"base\":${TARGET}, \"head\":${SOURCE}, \"draft\":${DRAFT}}"
+        DATA="{\"title\":${TITLE}, \"body\":${BODY}, \"base\":${TARGET}, \"head\":${SOURCE}, \"draft\":${DRAFT}, \"maintainer_can_modify\":${MAINTAINER_CAN_MODIFY}}"
         echo "curl --user ${GITHUB_ACTOR} -X POST --data ${DATA} ${PULLS_URL}"
```

Comment:
Where are you adding the assignee? I see it going into the function but not properly parsed.

Reply:
this is the maintainer_can_modify, the assignee in another curl call.

Reply:
Sorry, you misread the question. In the call to this function, you pass a variable for an ASSIGNEE but you never do anything with it.

```bash
 create_pull_request "${BRANCH}" "${PULL_REQUEST_BRANCH}" "${PULL_REQUEST_BODY}" "${PULL_REQUEST_TITLE}" "${PULL_REQUEST_DRAFT}" "${PULL_REQUEST_MAINTAINER_CAN_MODIFY}" "${PULL_REQUEST_ASSIGNEE}"
```
Note the last variable in that list - `PULL_REQUEST_ASSIGNEE` is not handled or passed to the call.